### ISSUE TITLE
Trezor | Wipe wallet adds a confusing 'Trezor wallet attached' alert on successful wipe

### ISSUE THREAD

**Issue by bitcoin-solutions** [bitcoin-solutions/multibit-hd#349]
See screenshot.
The wipe works correctly - it's just the alert that appears is confusing to users:

![screen shot 2015-01-12 at 15 40 56](https://cloud.githubusercontent.com/assets/5002074/5705482/94da5774-9a71-11e4-9e50-a1c8c2169fb0.png)

**Comment by bitcoin-solutions** [comment]
I have added into HardwareWalletContext the last time the device was wiped so MainController can suppress alerts due to the 'deviceReady' events that get fired, correctly, after a wipe.

Ready for review and closing

**Comment by gary-rowe** [comment]
I'm not convinced that putting an MBHD state dependency into MultiBit Hardware is the best solution to this. Could the deviceID from Features be used instead? I'll take a look at this tonight.

**Comment by bitcoin-solutions** [comment]
One thing to note with deviceId is that it is randomly generated every time a trezor is plugged in.
It's a bit of a gotcha as you cannot use it to persist state across Treezor usages.

**Comment by bitcoin-solutions** [comment]
if you can think of a cleaner way to do it then sure go for it!

**Comment by gary-rowe** [comment]
I opted to remove the "wipe time" from the MultiBit Hardware code since it breaks the principle of least surprise. While I was in there I noticed a couple of things that needed tidying up:

* device ID is definitely not suitable for "wipe time"
* entropy was not getting set after successfully completing the cipher key
* high and low level events needed to run asynchronously with better error reporting
* the client monitor thread is better structured as a scheduleWithFixedDelay 

So after sorting those out (now in staging repo) the problem of the "wipe time" became a  `ComponentChangedEvent` for the `MainController` to set a `lastWipeTime` field. I've tested it and it works fine (neatly leading into a whole create Trezor wallet scenario on re-insertion).

Ready for you to review and close now.
